crates/biome_js_syntax/src/expr_ext.rs

```diff
@@ -284,6 +290,22 @@ impl JsBinaryExpression {
             Ok(false)
         }
     }
+
+    pub fn get_node_position(&self, node: &JsSyntaxNode) -> Option<BinaryExpressionNodePosition> {
+        let node_name = node.text_trimmed().to_string();
```

Comment:
Could you avoid string alloc?

Reply:
@ematipico Suggested some different approach, i will refactor it